src/components/ChildIncidents/index.js

```diff
@@ -102,15 +103,25 @@ const Li = styled(ListItem)`
 
 const ChildIncidents = ({ className, incidents }) => (
   <List className={className} data-testid="childIncidents">
-    {incidents.map(({ href, status, values, changed }) => {
-      const valueEntries = Object.entries(values).map(([key, value]) => (
-        <DisplayValue key={key} title={value}>
-          {value}
-        </DisplayValue>
-      ));
+    {incidents.map(({ href, status, values: childIncident, changed }) => {
+      const valueEntries = (
+        <Fragment>
+          <DisplayValue key="id" title={childIncident.id}>{childIncident.id}</DisplayValue>
+
+          <DisplayValue key="category-and-status" title={`${childIncident.category} - ${childIncident.status}`}>
+            {childIncident.category}
+            <br />
+            {childIncident.status}
+          </DisplayValue>
+
+          <DisplayValue key="handling-time" title={childIncident.handlingTime}>
+            {childIncident.handlingTime}
+          </DisplayValue>
+        </Fragment>
+      );
 
       return (
-        <Li key={JSON.stringify(values)} status={status} numValues={values.length} changed={changed}>
+        <Li key={JSON.stringify(childIncident)} status={status} numValues={childIncident.length} changed={changed}>
```

Comment:
numValues is not used in Li so it can be removed

Reply:
This was already there, maybe could we use `props validation` on `styled components`  to prevent this in the future.

Removed it anyway ;)